SECURITY.md

```diff
@@ -0,0 +1,20 @@
+# Eiffel-community vulnerability and Security reporting
+
+## Report a Vulnerability or Security issue
+We as a community encourage researchers, users and contributors to report vulnerabilities and security related issues to the eiffel community. All issues are thoroughly investigated by a community security officer and/or other community security volunteers.
```

Comment:
Would be good to have a link to a page where all the reported/fixed issues are listed so people can be encouraged to check this before submitting a new issue?

The page could be linked to eiffel-community.github.io; something like https://eiffel-community.github.io/security.html

Reply:
Fixed